ktor-client/ktor-client-tests/common/test/io/ktor/client/tests/BodyProgressTest.kt

```diff
@@ -180,7 +180,7 @@ class BodyProgressTest : ClientLoader() {
             client.preparePost("$TEST_SERVER/content/echo") {
                 body = channel
                 onDownload(listener)
-            }.body<ByteReadChannel>()
+            }.body<ByteReadChannel, Unit> { it.readRemaining() }
```

Comment:
could you tell me why do we need second type here?

Reply:
The first type is for parameters, the second is for the return type of a block. It's an existing API and it wasn't changed in this PR.